plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/XmlValidatorTest.java

```diff
@@ -131,6 +144,24 @@ public void testValidate_withBannedElements() throws IOException, CoreException
   }
   
   @Test
+  public void testXsdValidation() throws CoreException {
+    String xml = "<?xml version='1.0' encoding='utf-8'?>"
```

Comment:
You don't really need the XML declaration in inline test data. It doesn't hurt but you can take it out and make the code a bit smaller.